js/store.js

```diff
@@ -53,6 +53,37 @@ export class StoreClass {
     return undefined
   }
 
+  getMetricDrawState (metricName) {
+    for (const metricBase in this.state.allMetrics) {
+      const metricArray = []
+      if (this.state.allMetrics[metricBase].name === metricName) {
```

Comment:
We really need a better way to get a metric by name other than looping through all metrics... But I guess this is a different refactoring issue.